### PR TITLE
Use same color for colored icon and text in menus, callouts

### PR DESCRIPTION
**Pull request by llorca**
With this PR, colored icons get the same color value than the text next to it, in menus, callouts... Somehow, this was already the case for minimal buttons, so nothing to change there.

Before:
image
image

After:
image
image

cc @pkwi

### CODE REVIEW

**Comment by pkwi** [comment]
Looking good!

**Comment by leebyp** [comment]
@llorca should input icons have the same colour change?

**Comment by llorca** [comment]
Oh yes missed those. @pkwi?

**Comment by pkwi** [comment]
yeah, same for colour for input icons

**Comment by llorca** [comment]
@pkwi input intents never have colored text, and the current icon color is shade 3:
image

Does it still need to be updated?

**Comment by pkwi** [comment]
I think they should get updated too. It makes sense to keep these icons consistent.

File: packages/core/src/components/forms/_input-group.scss [hunk 1]
```diff
@@ -197,7 +197,11 @@ $input-button-height-large: $pt-button-height !default;
       }
 
       .pt-icon {
-        color: $color;
+        color: map-get($pt-intent-text-colors, $intent);
+      }
+
+      .pt-dark & .pt-icon {
```
Comment (giladgray) [inline packages/core/src/components/forms/_input-group.scss]: move this inside the block above. `.pt-icon { .pt-icon & { ... } }`

**Pull request merged**